pkgs/by-name/mi/mingtest/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  cmake,
+  texliveSmall,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  name = "mingtest";
+  version = "0.1.9-4";
+  src = fetchFromGitHub {
+    owner = "craflin";
+    repo = "mingtest";
+    rev = "b6af8ada0908ea855a563ef43d5543bab8512d7d";
+    sha256 = "PSJ1+PtPR59qO/msCTS3o2ShjYkhO4h0Mtn0XY+KluQ=";
+  };
+
+  postPatch = ''
+    sed -i -e 's$include(CDeploy)$$' CMakeLists.txt
+    sed -i -e 's$install_deploy_export()$$' CMakeLists.txt
+  '';
+
+  buildInputs = [
+    cmake
+    texliveSmall
+  ];
+
```

Comment:
We need a `meta`, containing at least `description` `homepage` `license` `maintainers` `platforms`. See https://nixos.org/manual/nixpkgs/unstable/#chap-meta